configs/soft_teacher/semi_base_faster-rcnn_r50_caffe_fpn_180k_partial_coco.py

```diff
@@ -0,0 +1,70 @@
+_base_ = [
+    '../_base_/models/faster_rcnn_r50_fpn.py', '../_base_/default_runtime.py',
+    'semi_partial_coco_detection.py'
+]
+
+detector = _base_.model
+detector.data_preprocessor = dict(
+    type='DetDataPreprocessor',
+    mean=[103.530, 116.280, 123.675],
+    std=[1.0, 1.0, 1.0],
+    bgr_to_rgb=False,
+    pad_size_divisor=32)
+detector.backbone = dict(
+    type='ResNet',
+    depth=50,
+    num_stages=4,
+    out_indices=(0, 1, 2, 3),
+    frozen_stages=1,
+    norm_cfg=dict(type='BN', requires_grad=False),
+    norm_eval=True,
+    style='caffe',
+    init_cfg=dict(
+        type='Pretrained',
+        checkpoint='open-mmlab://detectron2/resnet50_caffe'))
+
+model = dict(
+    _delete_=True,
+    type='SemiBaseDetector',
+    detector=detector,
+    semi_train_cfg=dict(
```

Comment:
Can't it be called train_cfg?

Reply:
It aims to diff from `train_cfg` of `Faster RCNN` to avoid confusion confusion